golang/BUILD

```diff
@@ -102,4 +102,9 @@ intellij_integration_test_suite(
         "//third_party/go:go_for_tests",
         "@junit//jar",
     ],
+    env = {
+        # Disable runtime checks because GoGenericsLibraryRootsProvider causes them to trigger
+        # genericsStub folder containing builtin.go should be added to source roots, but it's not
+        "NO_FS_ROOTS_ACCESS_CHECK": "true"
```

Comment:
I don't know enough about the IntelliJ plug-in build setup to judge the impact of this, have to defer to the assigned reviewers here.

The rest looks good to me, thank you!

Reply:
Simply disabling this check looks a bit suspicious. Have you tried to adjust the test setup to add the necessary folder as source root?

Reply:
I agree it doesn't look clean. I tried a few things but without success. From decompiled code of `GoGenericsLibraryRootsProvider` I see that it's trying to access these folders in `getAdditionalProjectLibraries` method while the extension is loaded. So not much can be done on the configuration level. I've reported the issue to JetBrains folks though. They told me it's fixed but not released yet.

Reply:
Got news that it's fixed for 2022.1 and might be cherry picked to 2021.3.2.

Reply:
I asked @artspb in a private message. There is no YouTrack issue. Added the comment in 42ac35a916f10e04fa88622954ce54e3ffbf04e5

Reply:
We fixed the problem within [GO-12171](https://youtrack.jetbrains.com/issue/GO-12171) by removing `genericsStub`. The fix was back-ported to 2021.3 and will appear in 213.6777.*. It's expected to be released by the end of the month. I apologize for the inconvenience caused.